jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -19,40 +19,42 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
 class MachineView extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```

Comment:
We actually rarely pass props into the constructor as they're only required if you need to access the props in the constructor e.g. setting up state.